### PR TITLE
NETBEANS-3580] : Fixed Debug Expression Lambda 

### PR DESCRIPTION
**Pull request by singh-akhilesh**
https://issues.apache.org/jira/browse/NETBEANS-3580

### CODE REVIEW

File: java/debugger.jpda/src/org/netbeans/modules/debugger/jpda/breakpoints/LineBreakpointImpl.java [hunk 1]
```diff
@@ -337,32 +343,35 @@ protected void classLoaded (List<ReferenceType> referenceTypes) {
...
+                /* If location contains multiple event breakpoint for the same line 
+                (e.g lambda expression along with any function calling 
+                map(n -> foo(
+                        boo(n)))) 
+                then take the lowest location from each type of event*/
+                Collection<Location> lowestLocationList = locations
+                        .stream()
+                        .collect(groupingBy(loc->loc.method().toString(),
+                                LinkedHashMap::new,
+                                collectingAndThen(minBy(Comparator.comparingDouble(Location::codeIndex)),
```
Comment (entlicher) [inline java/debugger.jpda/src/org/netbeans/modules/debugger/jpda/breakpoints/LineBreakpointImpl.java:355]: It's better to use `Comparator.comparingLong`

Comment (singh-akhilesh) [reply java/debugger.jpda/src/org/netbeans/modules/debugger/jpda/breakpoints/LineBreakpointImpl.java]: comments addressed.

**Comment by arvindaprameya** [comment]
+1 Approved, thanks for fixing this Akhilesh

File: java/debugger.jpda/src/org/netbeans/modules/debugger/jpda/breakpoints/LineBreakpointImpl.java (hunk 1 above)
Comment (arusinha) [reply java/debugger.jpda/src/org/netbeans/modules/debugger/jpda/breakpoints/LineBreakpointImpl.java]: Please squash the changes before merging

**Comment by singh-akhilesh** [comment]
@ebarboni Please consider this PR to release 12.0

**Comment by neilcsmith-net** [comment]
The consideration should happen before merging - we're in freeze - if it's in master it's in 12.0!

**Comment by arusinha** [comment]
My mistake, I shouldn't had merged. I will tag Eric from next PR onward if I found it necessary for 12.0 and will not merge

**Comment by ebarboni** [comment]
No stress. Mistake happens, Would be nice to have a kind reminder on the github ui but ..

**Pull request merged**